tests/storage/blockvolume_test.py

```diff
@@ -73,7 +73,8 @@ class TestBlockVolumeSize(VdsmTestCase):
         # No initial size.
         # Expected 1024 MiB allocated (config.volume_utilization_chunk_mb)
         [(sc.SPARSE_VOL, sc.COW_FORMAT, (CONFIG.getint(
-            "irs", "volume_utilization_chunk_mb") - 1) * MiB, None), GiB],
+            "irs", "volume_utilization_chunk_mb") - 1) * MiB, None),
+         (CONFIG.getint("irs", "volume_utilization_chunk_mb") - 1) * MiB],
```

Comment:
I removed the constant because you seemed not convinced before, and after removing all the additional cases and test, my justification for having it (same as yours, more clear test data) felt not as relevant anymore.

I can bring it back, no problem. Lets will wait for the ost test to finish, and I will push the update.